src/worker-helpers.js

```diff
@@ -60,7 +60,7 @@ export function findESLintDirectory(modulesDir, config, projectPath) {
     eslintDir = Path.join(config.advancedLocalNodeModules || '', 'eslint')
   } else {
     locationType = 'advanced specified'
-    eslintDir = Path.join(projectPath, config.advancedLocalNodeModules, 'eslint')
+    eslintDir = Path.join(projectPath || __dirname, config.advancedLocalNodeModules, 'eslint')
```

Comment:
This most definitely isn't what you want here, if `null` is really getting legitimately passed in here, it should be handled differently.